docs/advanced_topics/api/v2/usage.md

```diff
@@ -0,0 +1,647 @@
+# Wagtail API v2 Usage Guide
+
+The Wagtail API module exposes a public, read only, JSON-formatted API which
+can be used by external clients (such as a mobile app) or the site's frontend.
+
+This document is intended for developers using the API exposed by Wagtail. For
+documentation on how to enable the API module in your Wagtail site, see
+[/advanced_topics/api/v2/configuration][1]
+
+[1]: https://github.com/wagtail/wagtail/blob/main/docs/advanced_topics/api/v2/configuration.md
+
+## Contents
+
+-   [Fetching content](#fetching-content)
```

Comment:
I believe this could be replaced by


    ```{contents}
    ---
    local:
    depth: 3
    ---
    ```

as per https://docs.wagtail.org/en/stable/contributing/documentation_guidelines.html#tables-of-contents

Reply:
The same was applied here.